ansible/aix61-standalone/manualBootstrap.md

```diff
@@ -9,19 +9,136 @@ modules will be installed.
 ```bash
 chfs -a size=+1300000 /opt
 chfs -a size=+250000 /
-chfs -a size=917504 /tmp
+chfs -a size=+917504 /tmp
```

Comment:
The original was intended, it sets /tmp to a size that is required for citgm. Using a + just makes it bigger, but there is no need to make it bigger if its big enough already,.

Reply:
I think I did this because that wasnt big enough but I cant remember why it wasnt big enough

Reply:
We should figure it out. The originals were those `+` statements, which were probably what was run, but doesn't help even a little bit when setting up new machines, where we need to know what the minimum required size is, not how much bigger the last machines disks needed to be!

I think I set the /tmp there so that all the machines would have the same /tmp size, so, should be big enough. If its not, then we need a new (absolute) size.